src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolStcicJudicialRoleMappingTest.java

```diff
@@ -34,25 +34,13 @@ class DroolStcicJudicialRoleMappingTest extends DroolBase {
 
     static {
         expectedRoleNameWorkTypesMap.put("senior-judge", null);
-        expectedRoleNameWorkTypesMap.put("judge", "hearing_work,decision_making_work,routine_work,"
-                + "applications,priority");
+        expectedRoleNameWorkTypesMap.put("judge", "decision_making_work");
         expectedRoleNameWorkTypesMap.put("case-allocator", null);
         expectedRoleNameWorkTypesMap.put("task-supervisor", null);
         expectedRoleNameWorkTypesMap.put("hmcts-judiciary", null);
         expectedRoleNameWorkTypesMap.put("specific-access-approver-judiciary", "access_requests");
         expectedRoleNameWorkTypesMap.put("leadership-judge", null);
-        expectedRoleNameWorkTypesMap.put("fee-paid-judge", "hearing_work,decision_making_work,"
-                + "routine_work,applications,priority");
```

Comment:
for roleAssignments that don't require worktypes, maybe we should add roleName to map with values set to null- e.g. `expectedRoleNameWorkTypesMap.put("senior-judge", null);` ?

Reply:
I have added all role names with null work types back to the map in the same order as the role mappings data.